google_drive/utils.py

```diff
@@ -1,37 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Dropped all of the utils?  Were they not actually common/stateless units of work?